### ISSUE TITLE
Impostor baking doesn't work on Apple Silicon (URP)

### ISSUE THREAD

**Issue by dev-fredericfox** [AmplifyCreations/AmplifyShaderEditor-Feedback#202]
```
[AmplifyImpostors] Something went wrong with the baking process, please contact support@amplify.pt with this log message.
Value cannot be null.
Parameter name: shader  at (wrapper managed-to-native) UnityEngine.Material.CreateWithShader(UnityEngine.Material,UnityEngine.Shader)
  at UnityEngine.Material..ctor (UnityEngine.Shader shader) [0x00008] in /Users/bokken/build/output/unity/unity/Runtime/Export/Shaders/Material.bindings.cs:18 
  at AmplifyImpostors.AmplifyImpostor.RenderAllDeferredGroups (AmplifyImpostors.AmplifyImpostorAsset data) [0x00c4f] in /Users/ImpostersDemo/Assets/AmplifyImpostors/Plugins/Scripts/AmplifyImpostor.cs:1250 
  at AmplifyImpostors.AmplifyImpostorInspector.DelayedBake () [0x00000] in /Users/ImpostersDemo/Assets/AmplifyImpostors/Plugins/Editor/Inspector/AmplifyImpostorInspector.cs:960 
UnityEngine.Debug:LogWarning (object)
AmplifyImpostors.AmplifyImpostorInspector:DelayedBake () (at Assets/AmplifyImpostors/Plugins/Editor/Inspector/AmplifyImpostorInspector.cs:965)
UnityEditor.EditorApplication:Internal_CallDelayFunctions () (at /Users/bokken/build/output/unity/unity/Editor/Mono/EditorApplication.cs:377)
```

Baking dosen't work on M1 macs.

**Comment by diogovtx** [comment]
I'll do my best to have a look at this ASAP. We're currently understaffed and looking to hire someone to help.

**Comment by Dawie3565** [comment]
Moving to Amplify Impostors v0.9.8.5
Note v0.9.8.4 was dealing with windows and Mac issues will follow shortly after in v0.9.8.5

**Comment by Farl** [comment]
Still stuck on similar issue in 0.9.8.5

Additional Info:
Unity Version: 2021.3
Machine: M1 Mac, M2 Mac, Intel Mac
Rendering Pipeline: URP 12
After Import "URP 12x.unitypackage" the message @dev-fredericfox mentioned won't show.
However, Editor will hang after press bake button.

**Comment by Dawie3565** [comment]
> 

yes we are working on Mac OS issue this week 
the last update was more for windows issues we needed to clear first to get Mac workflow moving forward

**Comment by diogovtx** [comment]
We had to release a hotfix, unrelated to Mac, for v0.9.8.5. The Mac fix is now highest priority and scheduled for v0.9.9. I'll be working on it tomorrow.

**Comment by Dawie3565** [comment]
Moved to Impostor Workflow 
https://github.com/AmplifyCreations/AmplifyImpostors-Feedback/issues

https://github.com/AmplifyCreations/AmplifyImpostors-Feedback/issues/17

@Farl @dev-fredericfox 
for info this is now fixed internally and just pending release in version 0.9.9.0 
i am just moving this from the ASE git to impostor git for record. 

thank you

**Comment by diogovtx** [comment]
> After Import "URP 12x.unitypackage" the message @dev-fredericfox mentioned won't show.

The issue reported by OP was caused by not importing the correct shaders. This will be addressed in v0.9.9 via new automatic importer, that selects the correct URP/HDRP shaders and reimports in the background, so it never happens again to anyone.

I also added a Start Screen to help with importing Samples.

> However, Editor will hang after press bake button.

This is the real problem and it is now fixed. I've tested the fix on URP and HDRP, 10 to 16 and it's working on all (tested on Apple Silicon).

A new build will be out today with all these fixes and improvements.

Thank you everyone for reporting.
